lib/pages/chat/events/message/message_content_with_timestamp_builder.dart

```diff
@@ -450,6 +453,42 @@ class _MessageContentWithTimestampBuilderState
                                                         .pop('pin');
                                                   },
                                                 ),
+                                                if (PlatformInfos
```

Comment:
please check this condition https://github.com/linagora/twake-on-matrix/blob/main/lib/pages/chat/chat.dart#L2094-L2109 and add relevant action in context menu